### PR TITLE
Added changelog

### PR DESCRIPTION
**Pull request by fredrikbryntesson**

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Frozen awaiting release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# ooc-kean changes by release
+
+## Release-x.x.x
+
+* Example 0
+	Explanation of Example 0
```
Comment (thomasfanell) [inline CHANGELOG.md]: I suggest we use this format instead:
- Example 0
  - Explanation of example 0

Comment (marcusnaslund) [inline CHANGELOG.md]: Or
```
- **Example 0**
Explanation of example 0
```

**Comment by marcusnaslund** [comment]
What's the status here?

**Comment by fredrikbryntesson** [comment]
yesyesyesyesyes

**Comment by marcusnaslund** [comment]
Fantastic.

**Pull request merged**